lesson31-if-loops/index.js

```diff
@@ -0,0 +1,251 @@
+//* ===== Minimum =====
+/* 1. Запитай у користувача його вік і визначи, ким він є: дитиною (0-11), 
+підлітком (12-17), дорослим (18_59) або пенсіонером (60 ...), передбач 
+можливість введення невірних даних.*/
+const age = prompt("Enter your age");
+
+if (age === null || age === " " || age === "") {
+  console.log("Please enter your age in numbers");
+} else {
+  Number(age);
+  switch (true) {
+    case age >= 0 && age <= 11:
+      console.log("You are children");
+      break;
+    case age >= 12 && age <= 17:
+      console.log("You are teenager");
+      break;
+    case age >= 18 && age <= 59:
+      console.log("You are adult");
+      break;
+    case age >= 60:
+      console.log("You are pensioner");
+      break;
+    default:
+      console.log("unrecognized");
+  }
+}
+
+/* 2. Запитай у користувача число від 0 до 9 і виведи йому спецсимвол, який розташований 
+на цій клавіші (1 !, 2 @, 3 # і т. д).*/
+let number = prompt("Enter any number from 0 to 9");
+
+if (number === null || number === " " || number === "") {
+  console.log(`${number} is not a number. Please enter number from 0 to 9.`);
+} else {
+  number = +number;
+  switch (number) {
+    case 0:
+      console.log(")");
+      break;
+    case 1:
+      console.log(`! - special character that is located on keyboard key 1`);
+      break;
+    case 2:
+      console.log("@ - special character that is located on keyboard key 2");
+      break;
+    case 3:
+      console.log("# - special character that is located on keyboard key 3");
+      break;
+    case 4:
+      console.log("$ - special character that is located on keyboard key 4");
+      break;
+    case 5:
+      console.log("% - special character that is located on keyboard key 5");
+      break;
+    case 6:
+      console.log("^ - special character that is located on keyboard key 6");
+      break;
+    case 7:
+      console.log("& - special character that is located on keyboard key 7");
+      break;
+    case 8:
+      console.log("* - special character that is located on keyboard key 8");
+      break;
+    case 9:
+      console.log(`( - special character that is located on keyboard key 9`);
+      break;
+    default:
+      console.log("unrecognized");
+  }
+}
+
+/* 3. Підрахуй суму всіх чисел в заданому користувачем діапазоні. */
+const userArray = [20, 40, 60];
```

Comment:
user should enter diapason like from 10 to 44
and you need to count each number between 10 and 44

Reply:
done